translations/en-us.all.json

```diff
@@ -1893,7 +1893,9 @@
   },
   {
     "id": "faq_ads_explanation",
-    "translation": "[lengthy explanation as to why we have ads"
+    "translation": "This is not a decision we make lightly, and we have thoroughly considered the objections raised by all of you, but ultimately we feel this is the only sustainable solution to fund Pantsu and ensure its continued operation. \
```

Comment:
In JSON format, you can't have line-breaks in string. You have to put every string in one line (only). That's how the format works. And that's why your travis fails

Reply:
https://stackoverflow.com/questions/2392766/multiline-strings-in-json